tests/resources/Harbor-Pages/Project-Helmcharts.robot

```diff
@@ -0,0 +1,63 @@
+*** Settings ***
+Documentation  This resource provides any keywords related to the Harbor private registry appliance
+Resource  ../../resources/Util.robot
+
+*** Keywords ***
+
+Switch To Project Charts
+    Click Element  xpath=//project-detail//a[contains(.,'Charts')]
+    Sleep  1
+    Page Should Contain Element  xpath=//hbr-helm-chart
+
+Upload Chart files
+    ${current_dir}=  Run  pwd
+    Run  wget ${harbor_chart_file_url}
+    Run  wget ${harbor_chart_prov_file_url}
+    Run  wget ${prometheus_chart_file_url}
+
+    Click Element  xpath=${upload_chart_button}
+    Choose File  xpath=${chart_file_browse}  ${current_dir}/${prometheus_chart_filename}
+    Click Element  xpath=${upload_action_button}
+    Sleep  2
+
+    Click Element  xpath=${upload_chart_button}
+    Choose File  xpath=${chart_file_browse}  ${current_dir}/${harbor_chart_filename}
+    Choose File  xpath=${chart_prov_browse}  ${current_dir}/${harbor_chart_prov_filename}
+    Click Element  xpath=${upload_action_button}
+    Sleep  2
+
+    Wait Until Page Contains  ${prometheus_chart_name}
+
+Go Into Chart Version
+    [Arguments]  ${chart_name}
+    Click Element  xpath=//hbr-helm-chart//a[contains(.,"${chart_name}")]
+    Sleep 2
+    Capture Page Screenshot  viewchartversion.png
+    Wait Until Page Contains  7.0.2
+
+Go Into Chart Detail
+    [Arguments]  ${version_name}
+    Click Element  xpath=//hbr-helm-chart-version//a[contains(.,"${version_name}")]
+    Sleep  2
+    Page Should Contain Element  ${chart_detail}
```

Comment:
Keyword like `Go Into xxx` should be reusable, so the verification should be moved to the test case level.